Lib/ctypes/util.py

```diff
@@ -80,6 +81,14 @@ def find_library(name):
                 continue
         return None
 
+if sys.platform.startswith("aix"):
+    # find .so members in .a files
+    # using dump loader header information + sys.
+    import ctypes._aix as aix
+
+    def find_library(name):
+        return aix.find_library(name)
```

Comment:
This can be written: find_library = aix.find_library, or simpler: from ctypes._aix import find_library

Reply:
Yes indeed!